Prac_05/commit4jack.py

```diff
@@ -0,0 +1,16 @@
+string = input("Enter a sentence ")
+counts = dict()
+words = string.split()
+
+longest = len(max(words, key=len))
+
+for word in words:
+    if word in counts:
+        counts[word] += 1
+    else:
+        counts[word] = 1
+
```

Comment:
There is an extra blank line here when there should only be one

Reply:
ok i will fix

Reply:
Thanks